src/caret_analyze/architecture/architecture.py

```diff
@@ -44,40 +47,43 @@ def __init__(
             file_type, file_path)
         loaded = ArchitectureLoaded(reader, ignore_topics)
 
-        self._nodes: Tuple[NodeStructValue, ...] = loaded.nodes
-        self._communications: Tuple[CommunicationStructValue, ...] = loaded.communications
-        self._executors: Tuple[ExecutorStructValue, ...] = loaded.executors
+        self._nodes: Tuple[NodeStruct, ...] = loaded.nodes
+        self._communications: Tuple[CommunicationStruct, ...] = loaded.communications
+        self._executors: Tuple[ExecutorStruct, ...] = loaded.executors
         self._path_manager = NamedPathManager(loaded.paths)
         self._verify(self._nodes)
 
     def get_node(self, node_name: str) -> NodeStructValue:
         try:
-            return Util.find_one(lambda x: x.node_name == node_name, self.nodes)
+            node: NodeStruct = Util.find_one(lambda x: x.node_name == node_name, self.nodes)
+            return node.to_value()
         except ItemNotFoundError:
             msg = 'Failed to find node. '
             msg += f'node_name: {node_name}'
             raise ItemNotFoundError(msg)
 
     def get_executor(self, executor_name: str) -> ExecutorStructValue:
-        return Util.find_one(lambda x: x.executor_name == executor_name, self.executors)
+        executor = Util.find_one(lambda x: x.executor_name == executor_name, self.executors)
+        return executor.to_value()
 
     def get_callback_group(self, callback_group_name: str) -> CallbackGroupStructValue:
-        return Util.find_one(
+        callback_group = Util.find_one(
             lambda x: x.callback_group_name == callback_group_name, self.callback_groups)
+        return callback_group.to_value()
 
     @property
     def callback_groups(self) -> Tuple[CallbackGroupStructValue, ...]:
         return tuple(Util.flatten([_.callback_groups for _ in self.executors]))
 
     @property
-    def callback_group_names(self) -> Tuple[CallbackGroupStructValue, ...]:
+    def callback_group_names(self) -> Tuple[str, ...]:
         return tuple(sorted(_.callback_group_name for _ in self.callback_groups))
 
     @property
     def topic_names(self) -> Tuple[str, ...]:
         return tuple(sorted({_.topic_name for _ in self.communications}))
 
-    def get_callback(self, callback_name: str) -> CallbackStructValue:
+    def get_callback(self, callback_name: str) -> CallbackStruct:
         return Util.find_one(lambda x: x.callback_name == callback_name, self.callbacks)
```

Comment:
こちら戻り値の型は変えないようお願いします。
```suggestion
    def get_callback(self, callback_name: str) -> CallbackStructValue:
        callback = Util.find_one(lambda x: x.callback_name == callback_name, self.callbacks)
        return callback.to_value()
```

Reply:
対応しました。
self.callbackの時点でTuple[CallbackStructValue]が返る実装になっているので、型指定だけ変更しました。